File: LeetCode-Problems/Easy/Two-Sum.java
//Java
class Solution {
    public int[] twoSum(int[] nums, int target) {
        int[] returnList = new int[2];
        for (int i = 0; i < nums.length; i++){
            for (int j = 0; j < nums.length; j++){
                if (i != j){
                    if (nums[i] + nums[j] == target){
                        returnList[0] = i;
                        returnList[1] = j;
                        return returnList;
                    }
                }
            }
        }
        return returnList;
    }
}

File: LeetCode-Problems/Easy/Two-Sum.py
# Python
class Solution(object):
    def twoSum(self, nums, target):
        """
        :type nums: List[int]
        :type target: int
        :rtype: List[int]
        """
        outputList = list()
        for x in range (0, len(nums)):
            for y in range (0, len(nums)):
                if (x != y):
                    if (nums[x] + nums[y] == target):
                        outputList.append(x)
                        outputList.append(y)
                        return outputList;
        return outputList;
        
